pkgs/development/python-modules/simplification/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  wheel,
+  numpy,
+  cython,
+  rdp,
+  pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "simplification";
+  version = "0.7.12";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "urschrei";
+    repo = "simplification";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DgPzcpg6x7kkNfOGypbdVys5iYMyLlf+SjOE1Mr7A+A=";
+  };
+
+  prePatch = ''
+    cp ${rdp}/{include/header.h,lib/librdp.so} src/simplification
+  '';
+
+  build-system = [ setuptools setuptools-scm wheel ];
+
+  dependencies = [ numpy cython ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = {
+    description = ''
+      Very fast Python line simplification using either the RDP or
+      Visvalingam-Whyatt algorithm implemented in Rust
+    '';
```

Comment:
should be one line

Reply:
The README says that it should be a single sentence and i cant see how to simplify this

Reply:
Then just put it in one line. There's no line length limit.